vaadin-dialog-flow-parent/vaadin-dialog-flow/src/main/java/com/vaadin/flow/component/dialog/Dialog.java

```diff
@@ -708,4 +717,14 @@ private void attachComponentRenderer() {
         setDimension(ElementConstants.STYLE_MIN_HEIGHT, minHeight);
         setDimension(ElementConstants.STYLE_MAX_HEIGHT, maxHeight);
     }
+
+    /**
+     * @throws UnsupportedOperationException
+     *             Adding styles to dialog overlay are not supported
```

Comment:
Let's align with the description of the same method in `ConfirmDialog`:

```suggestion
     *             Dialog does not support adding styles to overlay
```